internal/runners/config/meta.go

```diff
@@ -0,0 +1,23 @@
+package config
+
+import "github.com/ActiveState/cli/internal/constants"
+
+type event func() error
+
+type configType int
+
+const (
+	Int configType = iota
+	Bool
+)
+
+type configMeta struct {
```

Comment:
I struggled with the naming in this file. Open to suggestions.

Reply:
Maybe configRule, configSetup, or something similar.